runtime/dotnet/azurewebapp/Startup.cs

```diff
@@ -80,15 +80,16 @@ public IStorage ConfigureStorage(BotSettings settings)
             return storage;
         }
 
-        public BotFrameworkHttpAdapter GetBotAdapter(IStorage storage, BotSettings settings, UserState userState, ConversationState conversationState, IServiceProvider s)
+        public BotFrameworkHttpAdapter GetBotAdapter(IStorage storage, BotSettings settings, UserState userState, ConversationState conversationState, IServiceProvider s, TelemetryInitializerMiddleware telemetryInitializerMiddleware)
```

Comment:
Please port these to the `runtime/dotnet/azurefunctions` folder too. I can help you verify if you need.

Reply:
Added the changes to functions app as requested.